test-resources/infrastructure/template.yaml

```diff
@@ -32,25 +32,41 @@ Parameters:
     Default: common-cri-api
 
 Mappings:
+  KeyRotationMapping:
+    di-ipv-cri-address-api:
+      dev: true
+      build: true
+      staging: true
+      integration: false
+      production: false
+    di-ipv-cri-kbv-api:
+      dev: true
```

Comment:
Don't think KBV has a jwks endpoint yet that will be done in https://govukverify.atlassian.net/browse/OJ-3074 in sprint 36